db/migrate/20250512094010_add_ts_gender_to_people.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2025, Swiss Badminton. This file is part of
+#  hitobito_swb and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_swb.
+
+class AddTsGenderToPeople < ActiveRecord::Migration[7.1]
+  def change
+    add_column(:people, :ts_gender, :string, limit: 1)
+    reversible do |dir|
+      dir.up do
+        execute "UPDATE people SET ts_gender = gender"
+        execute "UPDATE people SET ts_gender = 'm' WHERE ts_gender IS NULL OR ts_gender = ''"
+      end
+    end
+    change_column_null(:people, :ts_gender, false)
```

Comment:
Das macht die Seeds kaputt, wenn auf der Datenbank NOT NULL verwendet wird, bräuchte es meiner Meinung nach einen default...

Ich denke eine presence validation auf dem Model wäre in diesem Fall besser

Reply:
die `presence` validierung kommt via `validates_by_schema` aufs model, somit braucht es keine explizite validierung

Reply:
Schon klar, nur gehen durch das NOT NULL auf der Datenbank bei einer Spalte ohne default value die Seeds kaputt: https://github.com/hitobito/hitobito_swb/actions/runs/14993589287/job/42122454544. Diese sollten noch gefixed werden